include/Reader/reader.h

```diff
@@ -136,21 +136,8 @@ extern "C"
     jitFilter(PEXCEPTION_POINTERS ExceptionPointersPtr, void *Param);
 extern void _cdecl fatal(int Errnum, ...);
 
-// Global environment config variables (set by GetConfigString).
-// These are defined/set in jit.cpp.
+// Environment config variables
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-extern uint32_t EnvConfigCseOn;
-#ifndef NDEBUG
-extern uint32_t EnvConfigCseBinarySearch;
-extern uint32_t EnvConfigCseMax;
-extern uint32_t EnvConfigCopyPropMax;
-extern uint32_t EnvConfigDeadCodeMax;
-extern uint32_t EnvConfigCseStats;
-#endif // !NDEBUG
 #if !defined(CC_PEVERIFY)
 extern uint32_t EnvConfigTailCallOpt;
```

Comment:
Yes, but we'll need to refactor the options object so it can be visible in the base reader (which is client agnostic). I'll open an issue.